spec/rubocop/cop/style/data_inheritance_spec.rb

```diff
@@ -0,0 +1,132 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Style::DataInheritance, :config do
+  context 'Ruby >= 3.2', :ruby32 do
+    it 'registers an offense when extending instance of Data' do
```

Comment:
Can you replace the entire test description with `Data.define`?

```suggestion
    it 'registers an offense when extending instance of `Data.define`' do
```